frontend/components/SmilePanel/components/WaysToEarnCard/index.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CheckIcon } from '@shopgate/engage/components';
+import BaseSmileLink from '../../../BaseSmileLink';
+import styles from './style';
+
+/**
+ * Ways to earn card
+ * @param {Object} option option info
+ * @returns {JSX}
+ */
+const WaysToEarnCard = ({ option }) => {
+  const { is_available, activity_rule } = option || {};
+  const cta = is_available ? null : (<CheckIcon className={styles.check} />);
+
+  return (
+    <BaseSmileLink
+      muted={!is_available}
+      iconImage={activity_rule.image_url}
```

Comment:
I didn't change this one since baseSmileLink is used with other components. Do you think it makes sense to update in the other component locations as well?